lib/exercism/rikki.rb

```diff
@@ -1,19 +1,27 @@
 require 'digest/sha1'
 
-class Rikki
-  def self.validate(key)
+module Rikki
+  extend self
```

Comment:
> I prefer to have the selfs be explicit, just because it's confusing to me otherwise.

Sounds good. I won't make the change from class to method then.